Added logs and timeout to Proofpoint Isolation (#39087)
demisto/content: Packs/ProofpointIsolation/ReleaseNotes/1_0_2.md

#### Integrations

##### Proofpoint Isolation

- Updated the Docker image to: *demisto/python3:3.12.8.1983910*.
- Added debug logs to the integration.

> - Added three retries to HTTP request for get_events in the Client class.

Reviewer: ```suggestion
- Added three retries to the HTTP request for get_events in the Client class.
```

> - Added three retries to the HTTP request for get_events in the Client class.

Reviewer 2: ```suggestion
-  Updated the request for get_events with the retries mechanism.
```
